code/modules/food/food/snacks_vr.dm

```diff
@@ -277,6 +277,57 @@
 	reagents.add_reagent("shockchem", 6)
 	bitesize = 7
 
+/obj/item/weapon/reagent_containers/food/snacks/lobster
+	name = "raw lobster"
+	desc = "a shifty lobster. You can try eating it, but its shell is extremely tough."
+	icon = 'icons/obj/food_vr.dmi'
+	icon_state = "lobster_raw"
+	nutriment_amt = 5
+
+/obj/item/weapon/reagent_containers/food/snacks/lobster/Initialize()
+	..()
+	bitesize = 0.1
+
+/obj/item/weapon/reagent_containers/food/snacks/lobstercooked
+	name = "cooked lobster"
+	desc = "a luxurious plate of cooked lobster, its taste accentuated by lemon juice. Reinvigorating!"
+	icon = 'icons/obj/food_vr.dmi'
+	icon_state = "lobster_cooked"
+	trash = /obj/item/trash/plate
+	nutriment_amt = 20
+	nutriment_desc = list("lemon" = 2, "lobster" = 5, "salad" = 2)
+
+/obj/item/weapon/reagent_containers/food/snacks/lobstercooked/Initialize()
+	..()
+	bitesize = 5
+	reagents.add_reagent("protein", 20)
+	reagents.add_reagent("tricordrazine", 5)
```

Comment:
The nutriment/protein amounts seem fine, but not really sure about having tricordrazine/iron. Tricordrazine is mainly a donk pocket/sin pocket thing, and Iron is mainly a liquidfood thing.

Reply:
Ah yeah, I forgot to describe that part. Intended to serve as something the chef can make for expiditions to virgo 4b. Though may need to increase the itemsize. Figured it'd be fine as it is more expensive/difficult to make than any ol food. you need to wait for cargo, and it costs cargo points.

But if you're that opposed, sure we can remove it.

Reply:
Yeah, remove the tricord/iron. Nutriment and protein are plenty beneficial on its own medically, and I would rather not completely obsolete donk/sin pockets.

Reply:
except you can easily get donk pockets and sin pockets anywhere. Why does junkfood heal but classy food doesn't ? 📦